cfme/tests/containers/test_blacklisted_container_events.py

```diff
@@ -69,6 +68,18 @@
 }
 
 
+def restore_advanced_settings(appliance):
+    """Restores the Advanced Settings config"""
+    if appliance.version < '5.10':
```

Comment:
Why not use `VersionPicker` here?

Reply:
I could but VersionPicker is generally used when you do not have the appliance object.